lib/openstack_query/utils.py

```diff
@@ -0,0 +1,53 @@
+from datetime import datetime, timedelta
+from exceptions.missing_mandatory_param_error import MissingMandatoryParamError
+
+
```

Comment:
Can we rename this to `time_utils` and maybe make a class of static methods, so that we can easily mock / DI it?